book/projects/project_readmes/dzdt.md

```diff
@@ -0,0 +1,26 @@
+# dzdt
+
+## Team Members
+
+The following people contributed to our project throughout the week:
+* Project lead: [Joanmarie Del Vecchio](url to their webpage)
+* Helper: [Scott Henderson](url to their webpage)
+* Helper: [Shashank Bushan](url to their webpage)
+* Team member: [Matt Olson](url to their webpage)
+
+## Project Goals
+
+In addition to learning more about working with ICESat-2 data on the cloud, our goals are to explore
+permafrost and land surface change with crossover ATL06 data
+
+## Project Outcomes
+
```

Comment:
* Created a GitHub Repository with Jupyter Notebooks detailing exploratory analysis at several areas of interest in the Arctic https://github.com/ICESAT-2HackWeek/dzdt

* Closed a SlideRule GitHub Issue related to raster sampling of new ArcticDEM v4.1 release (https://github.com/ICESat2-SlideRule/sliderule/issues/299#issuecomment-1674246119)

* Contributed to notes on 3D Geodetic transforms to inter-compare various elevation datasets https://github.com/ICESAT-2HackWeek/3D_CRS_Transformation_Resources